breacharbiter.go

```diff
@@ -585,34 +582,43 @@ justiceTxBroadcast:
 	err = b.cfg.PublishTransaction(finalTx, label)
 	if err != nil {
 		brarLog.Errorf("Unable to broadcast justice tx: %v", err)
+	}
```

Comment:
Hmm, shouldn't we now either terminate or attempt to further ascertain why the broadcast failed?

Reply:
The idea here was that it it fails, then it is most likely because of the `DoubleSpend` error, and we will retry the moment another tx spends any of the inputs. 

I do however see that we possibly should terminate if an unexpected error happens (to surface it), though it was not done before this commit either. What do you think?